archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/bash
+
+
+# Based on LordHeavy's LLVM
+pkgbase=llvm-git
+pkgname=('lldb-git' 'lld-git' 'polly-git' 'compiler-rt-git' 'clang-git' 'llvm-ocaml-git' 'llvm-libs-git' 'llvm-git'  'lib32-llvm-libs-git' 'lib32-clang-git' 'lib32-llvm-git') 
+pkgdesc='Low Level Virtual Machine (git version)'
+pkgver=12.0.0_r376876.6f0f0220380f
+pkgrel=1
+groups=('chaotic-mesa-git')
+arch=('x86_64')
```

Comment:
Please include armv7h and aarch64

Reply:
And in that case the `lib32` package should be conditional on `$CARCH = x86_64`.